test_runner/fixtures/neon_fixtures.py

```diff
@@ -1169,6 +1169,12 @@ def __init__(self, config: NeonEnvBuilder):
         if storage_controller_config is not None:
             cfg["storage_controller"] = storage_controller_config
 
+        if config.test_may_use_compatibility_snapshot_binaries:
+            if "storage_controller" in cfg:
```

Comment:
this could be better expressed via an if that assigns `{}`, then one unconditionally does `cfg["storage_controller"]["use_local_compute_notifications"] = False`.

Reply:
It's temporary code anyway so I'll just leave it as is.